test/test_view_ops.py

```diff
@@ -124,54 +124,132 @@ def test_conj_self(self, device, dtype):
         self.assertTrue(s is t)
 
     @onlyOnCPUAndCUDA
-    @dtypes(*get_all_fp_dtypes(include_bfloat16=False), torch.complex64)
-    def test_view_dtype(self, device, dtype):
-        int_dtype = {
-            torch.half: torch.int16,
-            torch.bfloat16: torch.int16,
-            torch.float: torch.int,
-            torch.double: torch.long,
-            torch.complex64: torch.long,
-        }[dtype]
-        numpy_dtype = {
-            torch.half: np.int16,
-            torch.bfloat16: np.int16,
-            torch.float: np.int32,
-            torch.double: np.int64,
-            torch.complex64: np.int64,
-        }[dtype]
+    @dtypes(*get_all_dtypes(include_bfloat16=False))
+    def test_view_dtype_new(self, device, dtype):
+        dtypes = {
```

Comment:
This can probably be imported from common_utils.py. See

https://github.com/pytorch/pytorch/blob/6011c35a7934a1528c2db02c40752aa3822ccfa6/torch/testing/_internal/common_utils.py#L760

and 

https://github.com/pytorch/pytorch/blob/6011c35a7934a1528c2db02c40752aa3822ccfa6/torch/testing/_internal/common_utils.py#L782